localuser/src/main/java/org/openmbee/sdvc/localuser/config/LocalUserSecurityConfig.java

```diff
@@ -55,6 +55,7 @@ public DaoAuthenticationProvider authenticationProvider() {
         try {
             userDetailsService.loadUserByUsername(adminUsername);
         } catch (UsernameNotFoundException e) {
+            userDetailsService.setPasswordEncoder(passwordEncoder());
```

Comment:
I think this was debug code. Can you confirm that this is required, and if not, remove?